README.md

```diff
@@ -31,6 +30,19 @@ steps:
   - run: oras version
 ```
 
+You can also download `oras` CLI from any preferred trusted source, by specifying link of the source via `url` and the SHA256 checksum via `checksum`. This can be used to download a `beta` or `rc` version, or download from a proxy. 
```

Comment:
```suggestion
You can also download `oras` CLI from any preferred trusted source, by specifying a link of the source via `url` and the SHA256 checksum via `checksum`. This can be used to download a pre-release version or download from a proxy other than `github.com`. 
```